cpp/unreal_plugins/SpServices/Source/SpServices/UnrealService.h

```diff
@@ -888,6 +888,14 @@ class UnrealService : public Service {
                 return toUInt64(UnrealClassRegistrar::createComponentOutsideOwnerConstructor(class_name, toPtr<AActor>(owner), component_name));
             });
 
+        unreal_entry_point_binder->bindFuncToExecuteOnGameThread("unreal_service", "create_component_outside_owner_constructor_by_class",
```

Comment:
- Either add `_by_class` variants consistently for all of the `create_component` entry points, or don't add it at all and just register your types ahead of time so you don't need a `_by_class` variant in your example apps. Either approach is fine with me.
- If you do add `_by_class` variants, lower them into `Unreal.h` and have `UnrealService.h` call your new `Unreal::` functions, similar to the other entry points in this file (e.g., `find_property_by_name`, etc).

Reply:
fixed